README.md

```diff
@@ -41,6 +41,8 @@ module.exports = config => {
           'test/my-test-case.js',
           'test/more-test-cases/*.js'
         ],
+        // You can also use a SharedWorker for test execution
+        worker: 'SharedWorker',
```

Comment:
Perhaps we should explain here that `'Worker'` is the default. Maybe something like?:

```javascript
// You can also use a SharedWorker for test execution,
// instead of the default 'Worker'
```